new/PRS12938Merrac.xml

```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12938Merrac" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Mǝrrāč̣</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="ES"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="ES" when="2020-03-31">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1"><roleName type="title">ደጅአዝማች፡ </roleName>ምራጭ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"><roleName type="title">dağğʾāzmāč</roleName> Mǝrrāč̣</persName>
+                    <persName xml:lang="gez" xml:id="n2" type="baptismal">ገብረ፡ እግዚአብሔር፡ </persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">Gabra ʾƎgziʾabǝḥer</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="political" cert="unknown">Could be the same as the early 20th-century
+                        governor of <placeName ref="LOC6145Wagari">Wagāriqo</placeName> (Liban, Eritrea).</occupation>
```

Comment:
I don't know the workflow behind these files. Does the governor have an ID? Are IDs for the family members foreseen?
If the aim is to quickly create crucial references for the important entities mentioned in E-S records, ignore these comments.

Reply:
For the moment it is only for the E-S records, yes, there is no alternative ID for the governor,  this is the only Merrac, the identification is taken from the MS record